enter_docker_env.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
```

Comment:
is it intentional to remove this file?

Reply:
Yeah, I shouldn't have checked it in in the first place.